ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -1341,6 +1341,24 @@ CHWZTC:
   implements:
   - CONTROL
 
```

Comment:
There's actually no DX in that FCU. It's a single chilled water valve for cooling that serves both zones